tests/loader_regression_tests.cpp

```diff
@@ -551,22 +551,98 @@ TEST(TryLoadWrongBinaries, WrongICD) {
     ASSERT_EQ(driver_count, 1);
 }
 
+TEST(TryLoadWrongBinaries, WrongExplicit) {
+    FrameworkEnvironment env{};
+    env.add_icd(TestICDDetails(TEST_ICD_PATH_VERSION_2));
+    env.get_test_icd().physical_devices.emplace_back("physical_device_0");
+
+    const char* layer_name_0 = "DummyLayerExplicit";
+    auto layer_0 = ManifestLayer{}.add_layer(
+        ManifestLayer::LayerDescription{}.set_name(layer_name_0).set_lib_path(CURRENT_PLATFORM_DUMMY_BINARY_WRONG_TYPE));
+
+    auto layer_loc_0 = env.explicit_layer_folder.write("dummy_test_layer_0.json", layer_0);
+    env.platform_shim->add_manifest(ManifestCategory::explicit_layer, layer_loc_0);
```

Comment:
Whoops, seems I need a 'is fake' flag that can be set for layers. Would prevent needing to manually talk to the platform_shim.

Reply:
I think it's ok for now.  Eventually.  We have too much else on our plate, and the framework allows us to test things we've never been able to do previously.  Let's add more tests and fix bugs and then revisit this.

Reply:
Agreed